jaxtronomy/Util/util.py

```diff
@@ -7,11 +7,10 @@
       (when applicable) and write test functions
 """
 
-import numpy as np
-import itertools
 from lenstronomy.Util.package_util import exporter
 import jax.numpy as jnp
 from jax import jit
+from lenstronomy.Util.util import make_grid, local_minima_2d, selectBest
```

Comment:
just checking, you want the lenstronomy and not jaxtronomy functions here

Reply:
Yes, I have not jaxified those yet. And I will not jaxify make_grid, may or may not jaxify the others